apps/coscrad-frontend/src/utils/generic-components/presenters/tables/generic-index-table-presenter/index-table.tsx

```diff
@@ -144,83 +152,107 @@ export const IndexTable = <T extends IBaseViewModel>({
         paginatedData.length === 0 ? (
             <NotFoundPresenter />
         ) : (
-            <div>
-                <table>
-                    <thead>
-                        <tr>
-                            {headingLabels.map(({ headingLabel }) => (
-                                <th key={headingLabel}>{headingLabel}</th>
-                            ))}
-                        </tr>
-                    </thead>
-                    <tbody>
-                        {paginatedData.map((row) => (
-                            <tr key={row.id} data-testid={row.id}>
-                                {headingLabels.map(({ propertyKey }) => (
-                                    // A little inversion of control here
-                                    // We may want to use some currying here
-                                    <td key={String(propertyKey)}>
-                                        {renderCell(row, cellRenderers, propertyKey)}
-                                    </td>
+            <Box sx={{ width: '100%' }}>
+                <Paper>
+                    <TableContainer>
+                        <Table aria-labelledby="Resources Table">
```

Comment:
If you want a border radius at the top like the table's footer you could add this,

```
                        <Table
                            aria-labelledby="Resources Table"
                            sx={{
                                borderCollapse: 'collapse',
                                borderRadius: '7px 7px 0 0',
                                overflow: 'hidden',
                            }}
                        >
```

Reply:
Thanks Justin.  I think there is a radius of 4px that comes with the Paper element.  We could give it a rounder 7px if we wanted.
<img width="761" alt="image" src="https://user-images.githubusercontent.com/63014783/232075082-79a4c761-bd92-4fd8-a270-65ed1804d844.png">